reference/orders.v3.yml

```diff
@@ -831,20 +863,63 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/MetafieldResponse'
+              examples:
+                example-1:
+                  value:
+                    data:
+                      permission_set: app_only
+                      namespace: 'minLength: 1'
+                      key: 'minLength: 1'
+                      value: 'minLength: 1'
+                      description: 'minLength: 0'
+                      resource_type: order
+                      resource_id: 'minimum: 0'
+                      id: 0
+                      created_at: '2019-08-24T14:15:22Z'
+                      updated_at: '2019-08-24T14:15:22Z'
+                    meta:
+                      meta:
+                        total: 36
+                        count: 36
+                        per_page: 50
+                        current_page: 1
+                        total_pages: 1
+                        links:
+                          previous: string
+                          current: '?page=1&limit=50'
+                          next: string
         '409':
           description: |
             The `Metafield` was in conflict with another `Metafield`. This can be the result of duplicate unique key combinations of the app's client id, namespace, key, resource_type, and resource_id.
```

Comment:
```suggestion
            The `Metafield` conflicted with another `Metafield`. This can result from duplicate unique key combinations of the app's client id, namespace, key, resource_type, and resource_id.
```